lib/main.dart

```diff
@@ -1,59 +1,54 @@
-// This example shows a [Scaffold] with an [AppBar], a [BottomAppBar] and a
-// [FloatingActionButton]. The [body] is a [Text] placed in a [Center] in order
-// to center the text within the [Scaffold] and the [FloatingActionButton] is
-// centered and docked within the [BottomAppBar] using
-// [FloatingActionButtonLocation.centerDocked]. The [FloatingActionButton] is
-// connected to a callback that increments a counter.
-
+import 'package:farmsmart_flutter/redux/app/app_state.dart';
+import 'package:farmsmart_flutter/redux/keys.dart';
+import 'package:farmsmart_flutter/redux/store.dart';
+import 'package:farmsmart_flutter/ui/home.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:redux/redux.dart';
 
-void main() => runApp(MyApp());
+void main() async{
+  // Defines app orientation
+  SystemChrome.setPreferredOrientations([
+    DeviceOrientation.portraitDown,
+    DeviceOrientation.portraitUp,
+  ]);
 
-class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
-  @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Flutter Code Sample for material.Scaffold',
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-      ),
-      home: MyStatefulWidget(),
-    );
-  }
+  debugDefaultTargetPlatformOverride = TargetPlatform.iOS;
+  var store = await createStore();
+  runApp(new FarmsmartApp(store));
 }
 
-class MyStatefulWidget extends StatefulWidget {
-  MyStatefulWidget({Key key}) : super(key: key);
+class FarmsmartApp extends StatefulWidget {
+  final Store<AppState> store;
+  FarmsmartApp(this.store);
 
+  // Async Creation of the App State
   @override
-  _MyStatefulWidgetState createState() => _MyStatefulWidgetState();
+  _AppState createState() => _AppState();
 }
 
-class _MyStatefulWidgetState extends State<MyStatefulWidget> {
-  int _count = 0;
-
+class _AppState extends State<FarmsmartApp> {
+  @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text('Sample Code'),
-      ),
-      body: Center(
-        child: Text('You have pressed the button $_count times.'),
-      ),
-      bottomNavigationBar: BottomAppBar(
-        child: Container(
-          height: 50.0,
-        ),
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: () => setState(() {
-              _count++;
-            }),
-        tooltip: 'Increment Counter',
-        child: Icon(Icons.add),
+    return StoreProvider<AppState>(
+      store: widget.store,
+      child: MaterialApp(
+          theme: ThemeData(
+            brightness: Brightness.light,
+            scaffoldBackgroundColor: Color(backgroundColor),
+            // ignore: strong_mode_invalid_cast_new_expr
```

Comment:
are these unused comments to be removed?

Reply:
Removed :)